Move ocp-qe-perfscale-ci repository from openshift-qe to openshift-eng org

paigerube14 opened the pull request:
https://issues.redhat.com/browse/DPP-16552
https://issues.redhat.com/browse/PERFSCALE-3739.

CC: @qiliRedHat

qiliRedHat commented:
@paigerube14 I had some comments above, would you check and see if there is any update needed?

paigerube14 commented:
> @paigerube14 I had some comments above, would you check and see if there is any update needed?

@qiliRedHat yes I updated on 1, 2 and 3. Going to remove references for 4 
The problem is here because I moved over the files. Any changes that go in now I won't have properly ported over so going to have to fully redo this work every time. we need to go ahead and move over sooner rather than later

qiliRedHat commented:
@paigerube14 I like to move as soon as possible. I asked in https://issues.redhat.com/browse/DPP-16552 a proper time for both you and Jone.

qiliRedHat commented:
@paigerube14 For 2 in my comment, the files are moved to the new folder but not updated. I saw the files contents are updated in another pr  https://github.com/openshift/release/pull/62152/files#diff-15d20f11a2923b986f8bfae9cc391484ee296fd3fb5aaf7bafe52be8723c9b89

paigerube14 commented:
@bradmwilliams @pruan-rht @bear-redhat @smg247 PTAL

bradmwilliams commented on core-services/release-controller/_releases/release-ocp-4.19.json:
```diff
@@ -505,7 +505,7 @@
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes"
+        "name": "periodic-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes"
```
Where is the actual job named: `periodic-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes` defined?  
The release-controller config failures you're getting are directly related to this question

paigerube14 replied:
talked with Brad offline and missed updating [this file](https://github.com/openshift/release/blob/129e601a900964133a1f47501f2fdbc433a018fe/hack/validate-release-jobs.py) where it defines which job files are used in release controller

bradmwilliams commented:
/label tide/merge-method-squash

bradmwilliams commented:
/approve
It won't mean much, but from the release-controller's perspective, I approve

pruan-rht commented:
/lgtm
/approve

paigerube14 commented:
/pj-rehearse ack

qiliRedHat commented:
/pj-rehearse reject

qiliRedHat commented:
/pj-rehearse ack

qiliRedHat commented:
/retest
